handler/src/test/java/io/netty/handler/ssl/OpenSslRenegotiateTest.java

```diff
@@ -21,6 +21,8 @@
 
     @Override
     public void testRenegotiateServer() throws Throwable {
+        // BoringSSL does not support renegotiate by purpose.
+        Assume.assumeFalse("BoringSSL".equals(OpenSsl.versionString()));
```

Comment:
Let's leave it as it is for now and fix this properly in netty-tcnative. We will notice this test starts to fail then. :-)